### DISCUSSION TITLE
How to keep input values but reset form to invalid

### DISCUSSION THREAD

**Discussion by cliffordp** [react-hook-form/react-hook-form#2441] [Q&A]
What I want:
Once the form validates, allow submission. Once submitted successfully, reset `formState.isValid` to `false` but leave all form field values unchanged to allow for quick and easy resubmission as long as one field is touched again (since validation is _onBlur_)

What I tried:
https://codesandbox.io/s/rhf-react-select-reset-invalid-after-submit-vrhgj?file=/src/index.js

**Comment by bluebill1049** [comment]
i am not sure if this is possible, because the form is still valid after reset with the same value.

**Reply by cliffordp** [reply]
I thought forcing invalid would then be valid again after touching any input since validation is `onBlur` -- so I just need them to blur something to help avoid sending the same submission again (although technically possible).

**Reply by bluebill1049** [reply]
What do you mean by forcing invalid?

**Reply by bluebill1049** [reply]
I thought you want a valid form after submit to become inValid but with valid form data. that's the bit confuses me.

**Reply by cliffordp** [reply]
right... but doesn't the validation happen onBlur? So imagine the async select field -- remember we had to force validate it as part of setValue()... -- imagine we're back to that point in time I guess, just prior to setValue's call to `{shouldValidate: true}`

**Reply by bluebill1049** [reply]
The form is already `valid` if you can submit and hit onSubmit right? there is no method right now to set the form to be `inValid` unless you manually change input to be invalid.
